dashboard/app/assets/stylesheets/application.scss

```diff
@@ -2014,6 +2015,10 @@ a.download-video {
   }
 }
 
+.container {
```

Comment:
Maybe worth a comment that this is overriding a value in an existing CSS class generated by Bootstrap?  (Also possibly worth changing in Bootstrap if we're generating a custom build of Bootstrap, though I'm not sure that we are.)